client/tracks/index.js

```diff
@@ -17,8 +17,8 @@ function isEnabled() {
 /**
  * Records site event.
  *
- * @param {string}  eventName       Name of the event.
- * @param {Object?} eventProperties Event properties.
+ * @param {string}  eventName       	Name of the event.
+ * @param {Object} [eventProperties]	Event properties (optional).
```

Comment:
I'm not sure about this change but it looks OK. Wondering why there are many spaces though.

Reply:
> I'm not sure about this change but it looks OK.

Thanks for checking! This was a bug – the old docblock said `eventProperties` was required, even though it's actually optional. My fix makes it actually optional. I had to do this to push my code 😁 

> Wondering why there are many spaces though

I was using tabs to align the "columns" in the docblock. I think that's normal style but happy to change it.